src/Workspaces/Core/Portable/Storage/SQLite/SQLitePersistentStorage.cs

```diff
@@ -197,7 +210,7 @@ private void CloseWorker()
             // are definitely persisted to the DB.
             try
             {
-                FlushAllPendingWritesAsync(CancellationToken.None).Wait();
+                FlushInMemoryDataToDisk();
```

Comment:
no task waits anymore :)

Reply:
note: we could even ignore this and only lose the last .5 seconds of writes.  something to consider if we want to avoid the io on shutdown.